### ISSUE TITLE
NoMethodError: undefined method `start' for nil:NilClass

### ISSUE THREAD

**Issue by mikz** [thoughtbot/capybara-webkit#903]
```
/Users/mikz/.rvm/gems/ruby-2.2.4/gems/capybara-webkit-1.10.0/lib/capybara/webkit/connection.rb:67:in `start_server'
/Users/mikz/.rvm/gems/ruby-2.2.4/gems/capybara-webkit-1.10.0/lib/capybara/webkit/connection.rb:17:in `initialize'
/Users/mikz/.rvm/gems/ruby-2.2.4/gems/capybara-webkit-1.10.0/lib/capybara/webkit/driver.rb:16:in `new'
/Users/mikz/.rvm/gems/ruby-2.2.4/gems/capybara-webkit-1.10.0/lib/capybara/webkit/driver.rb:16:in `initialize'
```

steps to reproduce:
```ruby
Capybara::Webkit::Driver.new(-> {})
```

I suspect https://github.com/thoughtbot/capybara-webkit/commit/753348ecb40b133e174225d8bd8b8ed900d2a2c2
because the driver creates copy of options, mutates the copy and passes the original along.
https://github.com/thoughtbot/capybara-webkit/blob/4eb066c92251dfaaf7649773183864caaf1fb0b2/lib/capybara/webkit/driver.rb#L14-L16

Imo `Connection.new` should get `@options` instead (which has the `:server` key).

I don't see how this could pass tests, because it fails to boot in default configuration.

**Comment by mikz** [comment]
Looks like there are 2 pull requests already: #901 and #902

**Comment by mhoran** [comment]
Fixed in capybara-webkit 1.10.1. Thanks!
